### DISCUSSION TITLE
Is it possible to correctly change the scale of the hand?

### DISCUSSION THREAD

**Discussion by MarchenkoSI** [leapmotion/UnityModules#1188] [Q&A] [answered]
Hello! Tell me, please, is it possible to correctly change the scale of the hand? When I change the scale, the elbow shifts and the arm stretches. 
![image](https://user-images.githubusercontent.com/12198147/164786653-a80b46ce-8028-4adb-a1ac-8509f625448b.png)

Plugin v 5.7.0
Unity 2020.3.26f1

**Comment by Ultraleap-Dan** [comment, marked as answer]
Please can you check to see if this has just been scaled for one axis? From your image it looks as though the model has been stretched along the X axis.

In addition please ensure that you have unticked the options below and this will allow to scale the hand successfully.

- Match joint postions with tracking data
- Scale model to tracking data

![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/59871906/166907278-3835f1c7-fb74-4a16-a38c-4fd1d8cba631.png)

Thank you for your feedback, we will look at improving our documentation for this to make it a bit clearer. :)
